src/pretalx/orga/forms/cfp.py

```diff
@@ -153,22 +156,45 @@ def __init__(self, *args, event=None, **kwargs):
         ):
             self.fields["is_public"].disabled = True
 
+    def clean(self):
+        deadline = self.cleaned_data["deadline"]
+        question_required = self.cleaned_data["question_required"]
+        if (deadline == "" or deadline is None) and (
+            question_required == "require after" or question_required == "freeze after"
+        ):
+            raise forms.ValidationError(
```

Comment:
Please make the errors translated strings using `_()`.

Reply:
🗺I fixed that for the two ValidationError messages in the file ```src/pretalx/orga/forms/cfp.py```.🗺